athenahealth/appointment_reminders.go

```diff
@@ -0,0 +1,79 @@
+package athenahealth
+
+import (
+	"context"
+	"net/url"
+	"strconv"
+	"time"
+)
+
+type ListAppointmentRemindersOptions struct {
+	StartDate    time.Time
+	EndDate      time.Time
+	DepartmentID string
+
+	AppointmentTypeID *int
+	PatientID         *int
+	ProviderID        *int
+	ShowDeleted       *bool
+
+	Pagination *PaginationOptions
+}
+
+type AppointmentReminder struct {
+	AppointmentReminderID int    `json:"appointmentreminderid"`
+	Note                  string `json:"note"`
+	Status                string `json:"status"`
+	Deleted               string `json:"deleted"`
+	PatientID             int    `json:"patientid"`
+	ProviderID            int    `json:"providerid"`
+	EncounterID           int    `json:"encounterid"`
+	DepartmentID          int    `json:"departmentid"`
+	ApproximateDate       string `json:"approximatedate"`
+	AppointmentTypeID     int    `json:"appointmenttypeid"`
+	StaffInstructions     string `json:"staffinstructions"`
+	PatientInstructions   string `json:"patientinstructions"`
+}
+
+type ListAppointmentRemindersResult struct {
+	Reminders []AppointmentReminder `json:"reminders"`
+
+	Pagination *PaginationResult
+}
+
+// ListAppointmentReminders - Retrieves a list of appointment reminders with the specified departmentid and
+// approximatedate within the given date range
+//
+// GET /v1/{practiceid}/appointments/appointmentreminders
+//
+// https://docs.athenahealth.com/api/sandbox#/Appointments/getPracticeidAppointmentsAppointmentreminders
```

Comment:
can we swap this with `https://docs.athenahealth.com/api/api-ref/appointment-reminders#Get-list-of-appointment-reminders`